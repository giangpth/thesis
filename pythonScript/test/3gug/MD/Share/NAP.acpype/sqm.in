Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
  15    PA        9.1480       -6.9540       27.8450 
   8   O1A       10.1160       -7.9600       27.3570 
   8   O2A        9.6220       -5.6270       28.3180 
   8   O5B        8.3080       -7.5700       29.0670 
   6   C5B        7.4730       -8.6870       28.9060 
   6   C4B        6.4840       -8.8350       30.0710 
   8   O4B        7.1650       -8.8810       31.3020 
   6   C3B        5.4510       -7.7210       30.2200 
   8   O3B        4.2470       -8.2380       29.7030 
   6   C2B        5.2880       -7.5110       31.7130 
   8   O2B        3.9650       -7.8180       32.2010 
   6   C1B        6.3140       -8.4390       32.3610 
   7   N9A        7.1690       -7.7700       33.3850 
   6   C8A        7.6230       -6.4920       33.3960 
   7   N7A        8.3710       -6.2870       34.4970 
   6   C5A        8.3880       -7.4650       35.1920 
   6   C6A        8.9700       -7.8940       36.3980 
   7   N6A        9.7440       -7.0920       37.1690 
   7   N1A        8.7740       -9.1750       36.7900 
   6   C2A        8.0250      -10.0180       36.0860 
   7   N3A        7.4440       -9.6630       34.9170 
   6   C4A        7.6240       -8.3840       34.4690 
   8    O3        7.9390       -6.6470       26.8050 
  15    PN        7.8820       -6.2290       25.2640 
   8   O1N        7.9790       -7.4330       24.3650 
   8   O2N        6.6430       -5.3680       25.1840 
   8   O5D        9.1640       -5.3150       24.9980 
   6   C5D        9.2710       -4.0360       25.6460 
   6   C4D        9.0520       -2.8760       24.6900 
   8   O4D       10.0270       -2.7940       23.6040 
   6   C3D        7.6680       -2.7840       24.0200 
   8   O3D        6.6170       -2.3780       24.9120 
   6   C2D        7.9400       -1.7380       22.9610 
   8   O2D        7.9760       -0.4240       23.5920 
   6   C1D        9.3460       -2.1940       22.4870 
   7   N1N        9.3290       -3.2030       21.3910 
   6   C2N        9.6930       -2.8060       20.1620 
   6   C3N        9.6770       -3.6850       19.0650 
   6   C7N       10.1240       -3.2170       17.7110 
   8   O7N        9.8550       -3.8930       16.7070 
   7   N7N       10.8630       -2.1010       17.7010 
   6   C4N        9.3110       -5.0280       19.2620 
   6   C5N        8.9460       -5.4450       20.5540 
   6   C6N        8.9730       -4.5060       21.6050 
  15   P2B        3.2100       -6.7330       33.1460 
   8   O1X        2.8310       -5.5710       32.2430 
   8   O2X        1.9920       -7.5460       33.5780 
   8   O3X        4.1470       -6.2920       34.2680 
   1     H       10.1410       -5.2010       27.6180 
   1    H1        8.0790       -9.5670       28.8530 
   1    H2        6.9090       -8.5520       28.0070 
   1    H3        5.9690       -9.7410       29.8270 
   1    H4        5.7220       -6.8110       29.7260 
   1    H5        4.0230       -9.0610       30.1660 
   1    H6        5.4360       -6.4790       31.9520 
   1    H7        5.7910       -9.2240       32.8650 
   1    H8        7.4250       -5.7840       32.6710 
   1    H9        9.9010       -6.1570       36.9040 
   1   H10       10.1500       -7.4450       37.9940 
   1   H11        7.8820      -10.9760       36.4430 
   1   H12        6.5210       -5.0610       24.2720 
   1   H13        8.5370       -3.9830       26.4230 
   1   H14       10.2640       -3.9520       26.0350 
   1   H15        9.1610       -2.0640       25.3780 
   1   H16        7.3110       -3.7240       23.6540 
   1   H17        6.8410       -1.5210       25.3070 
   1   H18        7.2230       -1.6580       22.1710 
   1   H19        8.6140       -0.4340       24.3230 
   1   H20        9.8320       -1.3190       22.1090 
   1   H21        9.9930       -1.8290       20.0180 
   1   H22       11.1910       -1.7360       16.8480 
   1   H23       11.0780       -1.6460       18.5470 
   1   H24       10.1430       -5.6290       18.9600 
   1   H25        8.4140       -5.1400       18.6890 
   1   H26        8.6630       -6.4220       20.7300 
   1   H27        8.7180       -4.8090       22.5580 
   1   H28        1.4640       -7.7760       32.7980 
   1   H29        4.8840       -5.7850       33.8930 

