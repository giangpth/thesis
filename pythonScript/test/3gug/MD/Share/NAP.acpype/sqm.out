            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =133

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: P : M.J.S.DEWAR et al. THEOCHEM, 187,1, (1989)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =      222
| QMMM:    diag iterations used for timing =        4
| QMMM:
| QMMM:              Internal diag routine = 0.565921 seconds
| QMMM:                 Dspev diag routine = 0.512627 seconds
| QMMM:                Dspevd diag routine = 0.521822 seconds
| QMMM:                Dspevx diag routine = 1.152397 seconds
| QMMM:                 Dsyev diag routine = 0.626773 seconds
| QMMM:                Dsyevd diag routine = 0.419995 seconds
| QMMM:                Dsyevr diag routine = 0.177961 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.162030 seconds
| QMMM:
| QMMM: Using dsyevr routine (diag_routine=7).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      P        9.1480   -6.9540   27.8450
  QMMM:     2        2      O       10.1160   -7.9600   27.3570
  QMMM:     3        3      O        9.6220   -5.6270   28.3180
  QMMM:     4        4      O        8.3080   -7.5700   29.0670
  QMMM:     5        5      C        7.4730   -8.6870   28.9060
  QMMM:     6        6      C        6.4840   -8.8350   30.0710
  QMMM:     7        7      O        7.1650   -8.8810   31.3020
  QMMM:     8        8      C        5.4510   -7.7210   30.2200
  QMMM:     9        9      O        4.2470   -8.2380   29.7030
  QMMM:    10       10      C        5.2880   -7.5110   31.7130
  QMMM:    11       11      O        3.9650   -7.8180   32.2010
  QMMM:    12       12      C        6.3140   -8.4390   32.3610
  QMMM:    13       13      N        7.1690   -7.7700   33.3850
  QMMM:    14       14      C        7.6230   -6.4920   33.3960
  QMMM:    15       15      N        8.3710   -6.2870   34.4970
  QMMM:    16       16      C        8.3880   -7.4650   35.1920
  QMMM:    17       17      C        8.9700   -7.8940   36.3980
  QMMM:    18       18      N        9.7440   -7.0920   37.1690
  QMMM:    19       19      N        8.7740   -9.1750   36.7900
  QMMM:    20       20      C        8.0250  -10.0180   36.0860
  QMMM:    21       21      N        7.4440   -9.6630   34.9170
  QMMM:    22       22      C        7.6240   -8.3840   34.4690
  QMMM:    23       23      O        7.9390   -6.6470   26.8050
  QMMM:    24       24      P        7.8820   -6.2290   25.2640
  QMMM:    25       25      O        7.9790   -7.4330   24.3650
  QMMM:    26       26      O        6.6430   -5.3680   25.1840
  QMMM:    27       27      O        9.1640   -5.3150   24.9980
  QMMM:    28       28      C        9.2710   -4.0360   25.6460
  QMMM:    29       29      C        9.0520   -2.8760   24.6900
  QMMM:    30       30      O       10.0270   -2.7940   23.6040
  QMMM:    31       31      C        7.6680   -2.7840   24.0200
  QMMM:    32       32      O        6.6170   -2.3780   24.9120
  QMMM:    33       33      C        7.9400   -1.7380   22.9610
  QMMM:    34       34      O        7.9760   -0.4240   23.5920
  QMMM:    35       35      C        9.3460   -2.1940   22.4870
  QMMM:    36       36      N        9.3290   -3.2030   21.3910
  QMMM:    37       37      C        9.6930   -2.8060   20.1620
  QMMM:    38       38      C        9.6770   -3.6850   19.0650
  QMMM:    39       39      C       10.1240   -3.2170   17.7110
  QMMM:    40       40      O        9.8550   -3.8930   16.7070
  QMMM:    41       41      N       10.8630   -2.1010   17.7010
  QMMM:    42       42      C        9.3110   -5.0280   19.2620
  QMMM:    43       43      C        8.9460   -5.4450   20.5540
  QMMM:    44       44      C        8.9730   -4.5060   21.6050
  QMMM:    45       45      P        3.2100   -6.7330   33.1460
  QMMM:    46       46      O        2.8310   -5.5710   32.2430
  QMMM:    47       47      O        1.9920   -7.5460   33.5780
  QMMM:    48       48      O        4.1470   -6.2920   34.2680
  QMMM:    49       49      H       10.1410   -5.2010   27.6180
  QMMM:    50       50      H        8.0790   -9.5670   28.8530
  QMMM:    51       51      H        6.9090   -8.5520   28.0070
  QMMM:    52       52      H        5.9690   -9.7410   29.8270
  QMMM:    53       53      H        5.7220   -6.8110   29.7260
  QMMM:    54       54      H        4.0230   -9.0610   30.1660
  QMMM:    55       55      H        5.4360   -6.4790   31.9520
  QMMM:    56       56      H        5.7910   -9.2240   32.8650
  QMMM:    57       57      H        7.4250   -5.7840   32.6710
  QMMM:    58       58      H        9.9010   -6.1570   36.9040
  QMMM:    59       59      H       10.1500   -7.4450   37.9940
  QMMM:    60       60      H        7.8820  -10.9760   36.4430
  QMMM:    61       61      H        6.5210   -5.0610   24.2720
  QMMM:    62       62      H        8.5370   -3.9830   26.4230
  QMMM:    63       63      H       10.2640   -3.9520   26.0350
  QMMM:    64       64      H        9.1610   -2.0640   25.3780
  QMMM:    65       65      H        7.3110   -3.7240   23.6540
  QMMM:    66       66      H        6.8410   -1.5210   25.3070
  QMMM:    67       67      H        7.2230   -1.6580   22.1710
  QMMM:    68       68      H        8.6140   -0.4340   24.3230
  QMMM:    69       69      H        9.8320   -1.3190   22.1090
  QMMM:    70       70      H        9.9930   -1.8290   20.0180
  QMMM:    71       71      H       11.1910   -1.7360   16.8480
  QMMM:    72       72      H       11.0780   -1.6460   18.5470
  QMMM:    73       73      H       10.1430   -5.6290   18.9600
  QMMM:    74       74      H        8.4140   -5.1400   18.6890
  QMMM:    75       75      H        8.6630   -6.4220   20.7300
  QMMM:    76       76      H        8.7180   -4.8090   22.5580
  QMMM:    77       77      H        1.4640   -7.7760   32.7980
  QMMM:    78       78      H        4.8840   -5.7850   33.8930

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -866.5769 kcal/mol        2.2911 kcal/(mol*A)
xmin    20     -873.5925 kcal/mol        1.7849 kcal/(mol*A)
xmin    30     -875.8744 kcal/mol        0.3283 kcal/(mol*A)
xmin    40     -877.4943 kcal/mol        0.3899 kcal/(mol*A)
xmin    50     -878.0244 kcal/mol        0.4393 kcal/(mol*A)
xmin    60     -878.5145 kcal/mol        0.4665 kcal/(mol*A)
xmin    70     -879.7403 kcal/mol        1.4135 kcal/(mol*A)
xmin    80     -881.8752 kcal/mol        0.5190 kcal/(mol*A)
xmin    90     -882.2564 kcal/mol        0.4546 kcal/(mol*A)
xmin   100     -882.5454 kcal/mol        0.1615 kcal/(mol*A)
xmin   110     -883.1541 kcal/mol        0.7669 kcal/(mol*A)
xmin   120     -883.9279 kcal/mol        0.2930 kcal/(mol*A)
xmin   130     -884.4990 kcal/mol        0.2721 kcal/(mol*A)
xmin   140     -884.8327 kcal/mol        0.1086 kcal/(mol*A)
xmin   150     -885.0975 kcal/mol        0.1494 kcal/(mol*A)
xmin   160     -885.4063 kcal/mol        0.1721 kcal/(mol*A)
xmin   170     -885.6344 kcal/mol        0.2176 kcal/(mol*A)
xmin   180     -885.8867 kcal/mol        0.4975 kcal/(mol*A)
xmin   190     -885.9978 kcal/mol        0.3132 kcal/(mol*A)
xmin   200     -886.1805 kcal/mol        0.1344 kcal/(mol*A)
xmin   210     -886.2820 kcal/mol        0.1529 kcal/(mol*A)
xmin   220     -886.3144 kcal/mol        0.1692 kcal/(mol*A)
xmin   230     -886.3331 kcal/mol        0.0442 kcal/(mol*A)
xmin   240     -886.4757 kcal/mol        0.4316 kcal/(mol*A)
xmin   250     -887.1553 kcal/mol        1.3466 kcal/(mol*A)
xmin   260     -890.2521 kcal/mol        0.3134 kcal/(mol*A)
xmin   270     -891.2708 kcal/mol        1.2334 kcal/(mol*A)
xmin   280     -892.5188 kcal/mol        0.3205 kcal/(mol*A)
xmin   290     -892.7103 kcal/mol        0.2955 kcal/(mol*A)
xmin   300     -892.8155 kcal/mol        0.2144 kcal/(mol*A)
xmin   310     -892.8902 kcal/mol        0.0991 kcal/(mol*A)
xmin   320     -892.9071 kcal/mol        0.0245 kcal/(mol*A)
xmin   330     -892.9125 kcal/mol        0.0277 kcal/(mol*A)
xmin   340     -892.9151 kcal/mol        0.0464 kcal/(mol*A)
xmin   350     -892.9170 kcal/mol        0.0176 kcal/(mol*A)
xmin   360     -892.9180 kcal/mol        0.0074 kcal/(mol*A)
xmin   370     -892.9181 kcal/mol        0.0033 kcal/(mol*A)
xmin   380     -892.9182 kcal/mol        0.0016 kcal/(mol*A)
xmin   390     -892.9182 kcal/mol        0.0016 kcal/(mol*A)
xmin   400     -892.9182 kcal/mol        0.0023 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -42.6948  -42.4797  -41.6447  -40.1598  -39.5384  -39.4660  -39.3199  -38.6951
 -38.6257  -38.3356  -38.0559  -38.0332  -37.9117  -37.7570  -37.5012  -37.2271
 -36.8623  -36.7973  -35.3010  -35.0730  -35.0458  -34.4745  -34.3977  -33.5880
 -32.8095  -32.2495  -32.1472  -31.6737  -30.6877  -30.6114  -29.8026  -27.4435
 -27.1017  -25.4356  -24.9226  -24.8531  -24.5943  -24.1604  -23.8177  -23.6522
 -23.2917  -22.8265  -21.9467  -21.6463  -21.2750  -21.1267  -20.5726  -20.2777
 -20.1341  -20.0053  -19.8616  -19.4905  -19.4392  -19.3996  -19.0981  -18.9740
 -18.6490  -18.4533  -18.1042  -18.0018  -17.9940  -17.5852  -17.4308  -17.3834
 -17.3158  -17.2741  -17.0368  -16.8988  -16.8159  -16.7055  -16.5187  -16.4547
 -16.4092  -16.2676  -16.2089  -16.1302  -15.9197  -15.8198  -15.7354  -15.6857
 -15.5276  -15.4388  -15.4087  -15.3179  -15.1218  -15.0340  -14.9343  -14.8552
 -14.7807  -14.7405  -14.6774  -14.4592  -14.4411  -14.2946  -14.2560  -14.1128
 -14.0870  -14.0804  -14.0237  -13.9352  -13.8188  -13.7218  -13.6912  -13.3089
 -13.2658  -13.2176  -13.1674  -13.0574  -12.9623  -12.8169  -12.6928  -12.5831
 -12.4752  -12.4380  -12.3506  -12.2337  -12.2119  -12.1934  -12.1585  -12.0897
 -11.9905  -11.9123  -11.6528  -11.5685  -11.4784  -11.1686  -10.8803  -10.4858
 -10.3530  -10.3492   -9.9111   -9.1211   -8.2658   -0.5038   -0.0979   -0.0768
   0.2933    0.6182    0.7127    0.9243    1.0527    1.0840    1.5205    1.5484
   1.7674    1.8296    2.1962    2.2344    2.2480    2.2638    2.2779    2.3161
   2.3722    2.3961    2.4632    2.4956    2.5476    2.6119    2.7538    2.8601
   2.9280    2.9567    2.9797    3.0115    3.0681    3.0875    3.1164    3.1354
   3.1619    3.1870    3.2124    3.3187    3.3669    3.3969    3.4703    3.5139
   3.5302    3.5401    3.5947    3.6262    3.6660    3.7532    3.9003    4.0115
   4.0222    4.1373    4.2300    4.3269    4.3766    4.4622    4.6125    4.6126
   4.7985    4.8959    4.9118    4.9243    4.9709    5.0107    5.0886    5.1184
   5.2822    5.3399    5.3578    5.3820    5.5256    5.5543    5.6142    5.6202
   5.6467    5.6562    5.9164    6.0486    6.0829    6.1091    6.1413    6.1495
   6.6961    6.7388    6.7423    6.9456    7.0299    7.7059

 Heat of formation   =       -892.91822134 kcal/mol  (      -38.71983961 eV)

 Total SCF energy    =    -239673.76068876 kcal/mol  (   -10393.03415675 eV)
 Electronic energy   =   -2355280.59689784 kcal/mol  (  -102132.63071410 eV)
 Core-core repulsion =    2115606.83620908 kcal/mol  (    91739.59655735 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      P                  2.798
     2      O                 -1.119
     3      O                 -0.808
     4      O                 -0.752
     5      C                  0.003
     6      C                 -0.043
     7      O                 -0.258
     8      C                  0.005
     9      O                 -0.338
    10      C                  0.007
    11      O                 -0.739
    12      C                  0.153
    13      N                 -0.216
    14      C                 -0.034
    15      N                 -0.112
    16      C                 -0.238
    17      C                  0.252
    18      N                 -0.351
    19      N                 -0.265
    20      C                  0.041
    21      N                 -0.268
    22      C                  0.073
    23      O                 -1.289
    24      P                  2.780
    25      O                 -1.082
    26      O                 -0.842
    27      O                 -0.735
    28      C                  0.030
    29      C                 -0.029
    30      O                 -0.339
    31      C                 -0.022
    32      O                 -0.305
    33      C                 -0.027
    34      O                 -0.304
    35      C                  0.136
    36      N                 -0.244
    37      C                 -0.004
    38      C                 -0.193
    39      C                  0.342
    40      O                 -0.383
    41      N                 -0.420
    42      C                 -0.070
    43      C                 -0.197
    44      C                 -0.069
    45      P                  2.629
    46      O                 -1.064
    47      O                 -0.766
    48      O                 -0.826
    49      H                  0.302
    50      H                  0.109
    51      H                  0.121
    52      H                  0.137
    53      H                  0.164
    54      H                  0.235
    55      H                  0.157
    56      H                  0.153
    57      H                  0.214
    58      H                  0.253
    59      H                  0.245
    60      H                  0.189
    61      H                  0.312
    62      H                  0.119
    63      H                  0.088
    64      H                  0.110
    65      H                  0.150
    66      H                  0.215
    67      H                  0.167
    68      H                  0.204
    69      H                  0.135
    70      H                  0.153
    71      H                  0.213
    72      H                  0.255
    73      H                  0.100
    74      H                  0.110
    75      H                  0.148
    76      H                  0.147
    77      H                  0.291
    78      H                  0.304
 Total Mulliken Charge =       0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -0.933    2.890    5.649    6.413
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      P        8.9110   -4.7982   26.7641
  QMMM:     2        2      O        9.8979   -5.5919   26.0613
  QMMM:     3        3      O        9.1440   -3.2414   26.8851
  QMMM:     4        4      O        8.6361   -5.2698   28.2682
  QMMM:     5        5      C        9.0142   -6.5685   28.6597
  QMMM:     6        6      C        7.7675   -7.4417   28.8666
  QMMM:     7        7      O        8.1559   -8.6224   29.5799
  QMMM:     8        8      C        6.6616   -6.7242   29.6627
  QMMM:     9        9      O        5.4562   -6.6771   28.9372
  QMMM:    10       10      C        6.5425   -7.4961   30.9978
  QMMM:    11       11      O        5.2846   -8.1063   31.0949
  QMMM:    12       12      C        7.6195   -8.6318   30.9065
  QMMM:    13       13      N        8.6871   -8.4941   31.8701
  QMMM:    14       14      C       10.0822   -8.3996   31.6289
  QMMM:    15       15      N       10.8153   -8.3613   32.7492
  QMMM:    16       16      C        9.8881   -8.4441   33.7994
  QMMM:    17       17      C       10.0305   -8.4539   35.2285
  QMMM:    18       18      N       11.2686   -8.3731   35.8000
  QMMM:    19       19      N        8.8951   -8.5523   36.0049
  QMMM:    20       20      C        7.7014   -8.6371   35.3671
  QMMM:    21       21      N        7.4161   -8.6443   34.0413
  QMMM:    22       22      C        8.5358   -8.5394   33.2644
  QMMM:    23       23      O        7.4116   -4.7885   26.1885
  QMMM:    24       24      P        5.8759   -4.9172   25.6925
  QMMM:    25       25      O        5.5005   -6.2370   25.2460
  QMMM:    26       26      O        5.1421   -4.3719   26.9849
  QMMM:    27       27      O        5.7210   -3.6766   24.7025
  QMMM:    28       28      C        5.8863   -2.3724   25.2118
  QMMM:    29       29      C        6.8479   -1.5627   24.3450
  QMMM:    30       30      O        8.1025   -2.2352   24.2156
  QMMM:    31       31      C        6.3027   -1.2839   22.9289
  QMMM:    32       32      O        5.6982   -0.0177   22.8500
  QMMM:    33       33      C        7.5269   -1.4689   22.0074
  QMMM:    34       34      O        8.1468   -0.2336   21.7236
  QMMM:    35       35      C        8.4894   -2.3892   22.8202
  QMMM:    36       36      N        8.5125   -3.7581   22.3887
  QMMM:    37       37      C        9.6677   -4.5240   22.6026
  QMMM:    38       38      C        9.7243   -5.8430   22.2985
  QMMM:    39       39      C       10.9734   -6.6313   22.4744
  QMMM:    40       40      O       11.4157   -7.3698   21.5680
  QMMM:    41       41      N       11.6516   -6.4974   23.6641
  QMMM:    42       42      C        8.5618   -6.5876   21.7392
  QMMM:    43       43      C        7.3280   -5.7625   21.7493
  QMMM:    44       44      C        7.3300   -4.4539   22.0758
  QMMM:    45       45      P        4.3386   -7.6995   32.3448
  QMMM:    46       46      O        4.7567   -6.4903   33.0260
  QMMM:    47       47      O        2.9104   -7.8115   31.6716
  QMMM:    48       48      O        4.2800   -9.0554   33.1654
  QMMM:    49       49      H        8.7014   -2.6661   26.2603
  QMMM:    50       50      H        9.5662   -6.4699   29.6368
  QMMM:    51       51      H        9.6879   -7.0268   27.8836
  QMMM:    52       52      H        7.3736   -7.8155   27.8799
  QMMM:    53       53      H        6.9262   -5.6441   29.8459
  QMMM:    54       54      H        5.0979   -7.5751   28.8827
  QMMM:    55       55      H        6.7062   -6.8036   31.8732
  QMMM:    56       56      H        7.1399   -9.6504   31.0334
  QMMM:    57       57      H       10.4701   -8.3788   30.6019
  QMMM:    58       58      H       12.0630   -8.2876   35.2160
  QMMM:    59       59      H       11.3746   -8.3628   36.7839
  QMMM:    60       60      H        6.8216   -8.7090   36.0438
  QMMM:    61       61      H        4.9485   -4.9732   27.7046
  QMMM:    62       62      H        4.8827   -1.8587   25.1891
  QMMM:    63       63      H        6.2710   -2.4247   26.2669
  QMMM:    64       64      H        7.0602   -0.5917   24.8778
  QMMM:    65       65      H        5.4868   -2.0168   22.6743
  QMMM:    66       66      H        6.3692    0.6340   22.6011
  QMMM:    67       67      H        7.2477   -1.8863   21.0037
  QMMM:    68       68      H        8.5576    0.0884   22.5382
  QMMM:    69       69      H        9.5478   -1.9868   22.7858
  QMMM:    70       70      H       10.5382   -3.9725   23.0026
  QMMM:    71       71      H       12.4130   -7.1089   23.8326
  QMMM:    72       72      H       11.1887   -6.1115   24.4550
  QMMM:    73       73      H        8.3922   -7.5279   22.3340
  QMMM:    74       74      H        8.8043   -6.9093   20.6870
  QMMM:    75       75      H        6.3896   -6.2710   21.4880
  QMMM:    76       76      H        6.4065   -3.8500   22.1148
  QMMM:    77       77      H        2.5449   -8.6924   31.5940
  QMMM:    78       78      H        4.9569   -9.2521   33.8086

           --------- Calculation Completed ----------

